app/views/orders/show.html.erb

```diff
@@ -1,10 +1,13 @@
 <h4>Order:</h4>
 <ul>
 	<ui>Total Price: $<%= @order.total_price %></ui>
-	<ui><%= @order.created_at %></ui>
+	<ui>Order Date: <%= @order.date %></ui>
+	<ui>Order Placed by: <%= @order.user.first_name + " " + @order.user.last_name %></ui>
```

Comment:
This would be a great opportunity to create a method called `full_name` to keep this line short and sweet.

Reply:
I absolutely agree! I'll go ahead and make a full_name method in the user model.